ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -111,6 +111,14 @@ public SpecVersion forMilestone(final SpecMilestone milestone) {
     return specVersions.get(milestone);
   }
 
+  public SpecVersion forMilestoneOrDefault(final SpecMilestone milestone) {
```

Comment:
@ajsutton this is the kind of logic i need for the api, not sure if this is the best spot... we'd ideally never use it unless its in api, maybe it should live in the Migrating class?